internal/controller/nnf_clientmount_controller.go

```diff
@@ -254,7 +260,23 @@ func (r *NnfClientMountReconciler) changeMount(ctx context.Context, clientMount
 			if err := os.Chown(clientMountInfo.MountPath, int(clientMount.Spec.Mounts[index].UserID), int(clientMount.Spec.Mounts[index].GroupID)); err != nil {
 				return dwsv1alpha2.NewResourceError("unable to set owner and group for file system").WithError(err).WithMajor()
 			}
+
+			// If we're setting permissions then we know this is only happening once.  Dump the
+			// servers resource to a file that can be accessed on the computes. Users can then
+			// obtain ost/mdt information.
+			// FIXME: decouple from SetPermissions?
+			if clientMount.Spec.Mounts[index].Type == "lustre" {
+				serversFilepath := filepath.Join(clientMountInfo.MountPath, lustreServersFilepath)
+				if err := r.dumpServersToFile(ctx, clientMount, serversFilepath); err != nil {
+					return dwsv1alpha2.NewResourceError("unable to dump servers resource to file on clientmount path").WithError(err).WithMajor()
+				}
+
+				if err := os.Chown(serversFilepath, int(clientMount.Spec.Mounts[index].UserID), int(clientMount.Spec.Mounts[index].GroupID)); err != nil {
```

Comment:
Can this go in the dumpServersToFile() function?

Reply:
Yes